File: Projects/TransportathonHackathon.Clients/angular-ui/src/app/models/response-models/transport-requests/getByCompanyAndCustomerTransportRequestResponse.ts
import { PaymentRequest } from '../../domain-models/paymentRequest';

export interface GetByCompanyAndCustomerTransportRequestResponse {
  id: string;
  transportTypeId: string;
  transportType: string;
  isPaid: boolean;
  customerFirstName: string;
  customerLastName: string;
  companyName: string;
  countryFrom: string;
  countryTo: string;
  cityFrom: string;
  cityTo: string;
  placeSize: string;
  approvedByCompany: boolean | null;
  isFinished: boolean;
  paymentRequest: PaymentRequest;
  createdDate: Date;
  updatedDate: Date;
  startDate: Date;
  finishDate: Date | null;
}

File: Projects/TransportathonHackathon.API/TransportathonHackathon.Application/Features/TransportRequests/Queries/GetByCompanyAndCustomer/GetByCompanyAndCustomerTransportRequestResponse.cs
using TransportathonHackathon.Domain.Entities;

namespace TransportathonHackathon.Application.Features.TransportRequests.Queries.GetByCompanyAndCustomer
{
    public class GetByCompanyAndCustomerTransportRequestResponse
    {
        public Guid Id { get; set; }
        public Guid TransportTypeId { get; set; }
        public string TransportType { get; set; }
        public bool IsPaid { get; set; }
        public string CustomerFirstName { get; set; }
        public string CustomerLastName { get; set; }
        public string CompanyName { get; set; }
        public string CountryFrom { get; set; }
        public string CountryTo { get; set; }
        public string CityFrom { get; set; }
        public string CityTo { get; set; }
        public string PlaceSize { get; set; }
        public bool? ApprovedByCompany { get; set; }
        public bool IsFinished { get; set; }
        public Guid CompanyId { get; set; }
        public Guid DriverId { get; set; }
        public string Brand { get; set; }
        public string Model { get; set; }
        public int ModelYear { get; set; }
        public PaymentRequest PaymentRequest { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime? FinishDate { get; set; }
    }
}
